core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-sidebar/dot-category-field-sidebar.component.html

```diff
@@ -32,14 +32,16 @@
             </div>
             <div class="flex-grow-1 category-field__categories">
                 @if (store.mode() === 'list') {
-                    <dot-category-field-category-list
-                        @fadeAnimation
-                        (itemChecked)="store.updateSelected($event.selected, $event.item)"
-                        (rowClicked)="store.getCategories($event)"
-                        [categories]="store.categoryList()"
-                        [isLoading]="store.isListLoading()"
-                        [selected]="store.selectedCategoriesValues()"
-                        [breadcrumbs]="store.breadcrumbMenu()" />
+                    @if (store.isListLoaded()) {
```

Comment:
Do we need a test for this?

Reply:
not really,are cover in the `dot-category-field-category-list` component.